src/AriaDisablingButton.js

```diff
@@ -8,7 +8,6 @@ type AriaDisablingButtonProps = {
     onClick: (evt: any) => void,
     disabled: boolean,
     className?: string,
-    disabledClassName: string,
```

Comment:
Please put this property back. We can still use a general CSS class in our global styles but provide it to the AriaDisabingButton instances as a property. This decouples the CSS class specifics from the component implementation.

Reply:
My bad, will do it now :)